tests/README.md

```diff
@@ -0,0 +1,51 @@
+# SAF Testing notes
+
+Here are some artifacts to assist with testing the SAF after it is deployed.
+
+Currently this is just a "smoke test" that runs internal to the OCP cluster. It delivers collectd data to the SAF amqp node and verifies that it can be seen in prometheus. This is intended to be usable for developers and TravisCI to validate our builds before merging changes to this repo.
```

Comment:
A bit nitty, but can you wrap the text to 80 chars?

In vim, just highlight in visual mode and press `gq`. At least that's how my vim is setup, not sure if that's a stock option :)